contrib/rbmt.toml

```diff
@@ -0,0 +1,20 @@
+# Configuration for rbmt (Rust Bitcoin Maintainer Tools)
+
+[test]
+# Examples to run with specific features enabled.
+# Format: "example_name:feature1 feature2"
+examples = [
+    "v0:std",
+    "v2:std",
+    "v2-separate-creator-constructor:std",
+]
+
+# Features to test with the conventional `std` feature enabled.
+# Tests each feature alone with std, all pairs, and all together.
+features_with_std = ["rand", "serde", "base64", "miniscript"]
+
+# Features to test without the `std` feature.
+# Tests each feature alone, all pairs, and all together.
+# Note: rand is a little weird until we can upgrade secp256k1 to v0.30.0
+# Note: miniscript is not included here as it has its own no-std handling
+features_without_std = ["serde", "base64"]
```

Comment:
Whip time again, sorry bro. In patch 99c1db89248924fed199746825b9bbf60bc86b6e you add empty arrays for `exact_features` and `features_with_no_std` then later in the PR they are removed. In general please don't make changes to a series of patches like this because it makes reviewers who review with there terminal (read: proper geeks) jump over to github to add a review suggestion only to find the change no longer exists. Each patch should be discreet both in what it does and in how it is reviewed. Just slap me for the sermons, I warned you I was in a state.